ts/features/wallet/paypal/component/__test__/PayPalCheckoutPspComponent.test.tsx

```diff
@@ -23,11 +28,32 @@ describe("PayPalCheckoutPspComponent", () => {
     expect(component).toMatchSnapshot();
   });
 
+  it(`edit label should be always pressable`, () => {
+    const pspName = "pspName123";
+    const mockEditLabelOnPress = jest.fn();
+    const component = render(
+      <PayPalCheckoutPspComponent
+        onEditPress={mockEditLabelOnPress}
+        fee={fee}
+        pspName={pspName}
+        privacyUrl={"https://io.italia.it"}
+      />
+    );
+    expect(component).not.toBeNull();
+    const editLabel = component.queryByTestId("editLabelTestID");
```

Comment:
⛏️ **nitpick**:  it is recommended to use `*byTestId` [only after the other queries don't work for your use case](https://testing-library.com/docs/queries/bytestid/).
The other queries match more closely how a user interacts with the UI. In this case, you can use the label's text `I18n.t("global.buttons.edit").toUpperCase()` and possibly remove [the testID](https://github.com/pagopa/io-app/pull/3718/files#diff-0ab0b85d4953718579c65cd322891e9ad79df612548294ca0805277a8cd74ffcR72) to avoid other people relying on it.

On a side note, I'd normally use `component.get*` when I'm testing for behavior and `component.query*` when I'm testing for existence. This makes the intent clear and reduces the code:

```typescript
// if editLabel doesn't exist, just fails here
const editLabel = component.getByTestId("editLabelTestID"); 
// no need for ifs, non-null assertions, ...
fireEvent(editLabel, "onPress");
```

Reply:
This is a TIL, for sure! thank you
007c53fb6a86757766492d74daf9626d539fbc6d